EntAppProject/src/main/resources/templates/JBCreateJobListing.html

```diff
@@ -19,12 +19,12 @@ <h2>Create A Job Listing</h2>
 <form action="/JBHomescreen">
 Description of Job:
 <br>
-<input style=position:left; type="text" id="userBio">
+<textarea style="width: 250px; height:100px;" id="jobDescription"placeholder="Job description here"></textarea>

```

Comment:
The change to textarea may be good, but there is still an inline style.  Change this to a linked style.